docker/catalog-auth-s3-otel/docker-compose.yml

```
@@ -108,6 +116,38 @@ services:
       timeout: 2s
       retries: 15
 
+  # Prometheus
+  prometheus:
+    image: prom/prometheus:v2.53.0
```

Please add `docker.io/` in front of all the image names - otherwise podman may ask to choose the "right" registry.
```suggestion
    image: docker.io/prom/prometheus:v2.53.0
```